src/types/models/account.ts

```diff
@@ -0,0 +1,11 @@
+export type Salutation = "mr" | "mrs" | "other"
+
+export interface UserAccount {
+  firstName: string
+  lastName: string
+  address: string
+  zipCode: number
+  city: string
+  phone: string
+  salutation: Salutation
```

Comment:
Maybe you can pick a set of selected keys from `BuyNowApplication`.

`Pick<Type, Keys>`